includes/blocks/sophi-page-list-item/index.css

```diff
@@ -0,0 +1,12 @@
+/**
+ * Example Block
+ * This file is for CSS admin overrides only.
+ * Use theme files for standard CSS work.
+ */
```

Comment:
Not sure this comment makes total sense in this context, or is needed

Reply:
https://github.com/globeandmail/sophi-for-wordpress/pull/333/commits/8cd22ada9fe3b5cc7ae8f271e064d16cdf12bf6f